<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logtable</class>
 <widget class="QWidget" name="logtable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>被查看用户：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUser"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExport">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonImport">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBoxCheckType">
       <item>
        <property name="text">
         <string>登陆时间</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>退出时间</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>开始擦除</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>擦除完成</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>擦除失败</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>磁盘插入</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>磁盘拔出</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEditStart">
       <property name="displayFormat">
        <string>yyyy-MM-dd  hh:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEditEnd">
       <property name="displayFormat">
        <string>yyyy-MM-dd  hh:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>介质类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>密级：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxLevel">
       <item>
        <property name="text">
         <string>机密</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>秘密</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绝密</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonQuery">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>至</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="logtableWidget">
     <column>
      <property name="text">
       <string>时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>密级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>原用途</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>介质去向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>审批人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>部门</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>责任人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>介质</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>硬盘序列号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作用户</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>用户等级</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
